### DISCUSSION TITLE
No 'Outbound' Monitored Network Traffic in My Security Onion 2.3 Install

### DISCUSSION THREAD

**Discussion by Katanguero** [Security-Onion-Solutions/securityonion#7343] [Unsupported Versions] [answered]
I installed the latest Security Onion 2.3.  I ran so-status and everything indicates that is working correctly.  I am able to see alerts and traffic. However, I got two things that I need to resolve:

1. I am not able to see the 'outbound' monitored traffic so I am only getting half of the conversations.  This is a problem because I don't know if my devices are actually acknowledging any request/conversations.
2. I only see the IP of my local ISP provider.  No internal (private) IPs are showing so I don't know what device is doing what.

My current set up is as follow:

cable modem to network tap.  Network tap to Google Wi-Fi router. Router to switch.  SO connected to network tap's monitoring jack and management NIC is connected to the switch.

I have researched for solutions and I am coming out empty.  Should I place the network tap to the router instead.  I am using a Datacom 10/100/1000 Network Tap.  Any help is appreciated.

**Comment by dougburks** [comment, marked as answer]
Please see this image from https://docs.securityonion.net/en/2.3/introduction.html:
![image](https://user-images.githubusercontent.com/1659467/155987002-910ff40b-f0d5-44ea-a060-64c68efda96c.png)

Notice that we recommend placing the TAP inside the firewall/NAT boundary. In your case, that would mean placing the tap between the Google Wi-Fi router and the switch.
